### ISSUE TITLE
Access to column data when __slot is used

### ISSUE THREAD

**Issue by ArturLew** [ratiw/vuetable-2#302]
Hi vuetable maniacs ;)

I'm using __slot mechanism to display some dynamic columns based upon the data from the service.

For every column from the service I use something like this:
this.fields.push({
          name: '__slot:res-col',
          title: `<div>${x}</div>`
})

For every dynamic column I've used the same slot: res-col
The problem is that in template I cannot access to column data (additional props or index).
According to documentation only: row-data, row-index and row-field are passed to the slot template.

I think that it's a good idea to improve above mechanism by introducing another property:
* col-index

It would allow to easy access to column information based on index.

**Comment by zbyte64** [comment]
Indeed. Right now it passes `field="field.sortField"` to slots and components. Seems strange that this value only gets passed along if the field is sortable.
